src/pytorch_lightning/loggers/tensorboard.py

```diff
@@ -224,6 +239,12 @@ def log_hyperparams(
 
         if metrics:
             self.log_metrics(metrics, 0)
+
+            if _TENSORBOARD_AVAILABLE:
+                from torch.utils.tensorboard.summary import hparams
+            else:
+                from tensorboardX.summary import hparams  # type: ignore[no-redef]
```

Comment:
how many times is this called per epoch?

Reply:
This was answered in https://github.com/Lightning-AI/lightning/pull/15762#discussion_r1028655601